src/lib/util.ts

```diff
@@ -14,6 +14,8 @@ const polyfills = {
 export const btoa = globalThis.btoa ?? polyfills.btoa;
 
 export const contractAddresses: { [index: string]: string } = {
+  local: "0x5fbdb2315678afecb367f032d93f642f64180aa3", // If building locally you can put your contract address here
```

Comment:
I added some more potential contracts and networks.  As we add deploy more contract+validator+gateways we can keep adding to this

Reply:
It is also possible to _supply_ this from user land, right?

Reply:
Yea, it will be after #45 
My thinking was that if we get to a place where running a fully local env is possible, having this will stream line things